### ISSUE TITLE
[Enhancement] support for iOS 15+ @Environment(\.dismiss)

### ISSUE THREAD

**Issue by K1NXZ** [shaps80/SwiftUIBackports#8]
In iOS 15+ the way to dismiss a sheet/modal is to use  [`@Environment(\.dismiss)`](https://developer.apple.com/documentation/swiftui/environmentvalues/dismiss). In previous versions it was [`@Environment(\.presentationMode)`](https://developer.apple.com/documentation/swiftui/presentationmode) which is now deprecated.

I am not sure what would be the best approach to implement the API in the Backport for that, any idea?

**Comment by shaps80** [comment]
Yep I actually have that on my list. Not difficult at all. It’s almost done cause I previously did my own presentation dismiss for standard UIKit presentation. I’ll get this in for sure 👍

**Comment by shaps80** [comment]
This is now completed and will be released later tonight.

**Comment by shaps80** [comment]
Closing, as this is now included in version `1.2.0`
